src/components/DivingCylinderSet/NewDivingCylinderSet.tsx

```diff
@@ -97,16 +98,24 @@ export const NewDivingCylinderSet = (): JSX.Element => {
       postDivingCylinderSet(payload),
     onSuccess: (cylinderSet) => {
       const cylinderSets = queryClient.getQueryData<DivingCylinderSet[]>(
-        CYLINDER_SETS_QUERY_KEY(userId)
+        DIVING_CYLINDER_SETS_QUERY_KEY(userId)
       );
 
-      queryClient.setQueryData(CYLINDER_SETS_QUERY_KEY(userId), [
+      queryClient.setQueryData(DIVING_CYLINDER_SETS_QUERY_KEY(userId), [
         ...(cylinderSets ?? []),
         cylinderSet,
       ]);
       toast.success('Uusi pullosetti lisätty!');
     },
-    onError: () => {
+    onError: (res: AxiosError) => {
+      // TODO: Figure out if this can be checked in validation
+      if (res.response?.status === 409) {
+        toast.error(
+          'Pullosetin nimi on jo käytössä. Valitse toinen nimi ja yritä uudelleen.'
+        );
+        return;
+      }
```

Comment:
Vaikkei constraintia enää olekkaan, halutaanko ohjata käyttäjää kuitenkin käyttämään uniikkeja nimiä näkyvillä olevien pullojen suhteen?

Reply:
Nyt kun se constraintti tosiaan on otettu pois niin ton endpointinhan ei enää pitäisi pystyä palauttamaan 409, joten mun mielestä sitä ei täälläkään pidä käsitellä erikseen. Se pitäis tosiaan poistaa sielä backendin puolelta. Teen siitä tiketin.

Reply:
https://github.com/Tampereen-Urheilusukeltajat/Blenderi-Backend/issues/68